pynetbox-stubs/core/api.pyi

```diff
@@ -9,10 +9,7 @@ from pynetbox._gen.tenancy import TenancyApp
 from pynetbox._gen.users import UsersApp
 from pynetbox._gen.virtualization import VirtualizationApp
 from pynetbox._gen.wireless import WirelessApp
-from pynetbox.core.app import App
-
-class PluginsApp:
-    pass
+from pynetbox.core.app import App, PluginsApp
 
```

Comment:
missing empty line: did you run `tox -e fixlint`?

Reply:
`missing empty line: did you run tox -e fixlint?`

tox is throwing loads of errors about all the other files in the repo (files I haven't touched), specifically black and isort, so it was tricky to filter out that stuff. I didn't want to raise a PR with changes to those other files (I will do this in a separate PR though)... So it seems I missed this in all the tox noise. Have fixed now.

Reply:
interesting. thanks for this